lib/kumascript/conf.js

```diff
@@ -0,0 +1,66 @@
+var _ = require('underscore'),
+    fs = require('fs'),
+    nconf = require('nconf');
+
+// ### Default configuration settings
+// This could go in `kumascript_settings.json` in the parent project.
+var DEFAULT_CONFIG = {
+    log: {
+        console: false,
+        file: {
+            filename: 'kumascript.log',
+            maxsize: 1024 * 100, // 100k
+            maxFiles: 5
+        }
+    },
+    statsd: {
+        enabled: false,
+        host: '127.0.0.1',
+        port: 8125
+    },
+    server: {
+        port: 9080,
+        numWorkers: 4,
+        workerConcurrency: 4,
+        workerTimeout: 1000 * 60,
+        workerMaxJobs: 8,
+        workerRetries: 3,
+        document_url_template:
+            "https://developer.mozilla.org/en-US/docs/{path}?raw=1",
+        template_url_template:
+            "https://developer.mozilla.org/en-US/docs/en-US/Template:{name}?raw=1",
+        template_cache_control: 'max-age=3600'
+    },
+    repl: {
+        enabled: true,
+        host: "127.0.0.1",
+        port: 9070
+    }
```

Comment:
I think we should have sane defaults for newrelic here, so values for `app_name`, `high_security` and `license_key` -- for the latter an empty string or something that we can check for in `server.js`.